lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -505,6 +506,26 @@ def commit(method, url, parameters = nil, options = {})
         )
       end
 
+      def add_level_iii_data(post, options)
+        # Hard coded, no validations, just to see that it gets added
+        post[:level3] = {}
+        post[:level3][:merchant_reference] = 55823
+        post[:level3][:shipping_address_zip]=94110
+        post[:level3][:shipping_amount]=0
+
+        # This one produces the following error:
+        # "Array must contain only hashes"
+        post[:level3][:line_items] = [{
+            :product_code => 0
+        }]
+
+        # This one produces the following error:
+        # "Invalid keys: all first indices must be numeric, you provided product_code"
+        # post[:level3][:line_items] = {
+        #     :product_code => 0
+        # }
+      end
```

Comment:
It seems that I'm unable to add the line items here. 

The format in the Stripe docs has an empty array in the middle and I can't get the format right
<img width="878" alt="screen shot 2016-08-20 at 3 55 28 pm" src="https://cloud.githubusercontent.com/assets/9390255/17834295/93cd776c-66ee-11e6-9329-41b66f37146e.png">